assets/patterns/header/README.md

```diff
@@ -0,0 +1,56 @@
+# Header
+
+{{ wip(4) }}
```

Comment:
Will this still be WIP after this gets merged?

Reply:
I would take it off.